File: class24/python/AP2401-Exemplo01.py
# ValidaEmail — Aula 24
# Funcao que valida formato basico de email

def validar_email(email):
    return "@" in email and "." in email.split("@")[-1]

email = input("Digite seu email: ")
if validar_email(email):
    print(f"Email {email} eh VALIDO!")
else:
    print(f"Email {email} eh INVALIDO!")

File: class24/cpp/AP2401-Exemplo01.cpp
#include <iostream>
#include <string>
bool validarEmail(std::string email) {
    size_t arroba = email.find(64);
    if (arroba == std::string::npos) return false;
    return email.find(46, arroba) != std::string::npos;
}
int main() {
    std::string email;
    std::cout << "Digite seu email: "; std::getline(std::cin, email);
    if (validarEmail(email)) std::cout << "Email " << email << " eh VALIDO!" << std::endl;
    else std::cout << "Email " << email << " eh INVALIDO!" << std::endl;
    return 0;
}